### PR TITLE
Add dev env

### PR DESCRIPTION
**Pull request by svrana**

### CODE REVIEW

File: internal/utils/connection_utils.go [hunk 1]
```diff
@@ -7,18 +7,19 @@ import (
...
+	"github.com/nucleuscloud/cli/internal/config"
+	clienv "github.com/nucleuscloud/cli/internal/env"
 )
 
 func getApiUrl() string {
 	if clienv.IsDevEnv() {
-		return "localhost:50051"
+		return "mgmt-api-nucleus.svcs.nucleuscloud.dev:443"
 	} else if clienv.IsStageEnv() {
 		return "mgmt-api-nucleus.svcs.stage.nucleuscloud.com:443"
```
Comment (nickzelei) [inline internal/utils/connection_utils.go:24]: what's up with this diff? 
https://github.com/nucleuscloud/cli/blob/main/internal/utils/connection_utils.go#L23

File: internal/utils/connection_utils.go [hunk 2]
```diff
@@ -7,18 +7,19 @@ import (
...
+
+	"github.com/nucleuscloud/cli/internal/auth"
+	"github.com/nucleuscloud/cli/internal/config"
+	clienv "github.com/nucleuscloud/cli/internal/env"
 )
 
 func getApiUrl() string {
 	if clienv.IsDevEnv() {
-		return "localhost:50051"
+		return "mgmt-api-nucleus.svcs.nucleuscloud.dev:443"
```
Comment (nickzelei) [inline internal/utils/connection_utils.go:22]: why is dev pointing to prod? on the wrong port?

Comment (svrana) [reply internal/utils/connection_utils.go:22]: read mo closely

Comment (nickzelei) [reply internal/utils/connection_utils.go:22]: ooooooooo.
why is it pointing to 443 though and not 50051?

**Review by nickzelei — APPROVED** [review]

**Pull request merged**